### PR TITLE
Implemente client.stream()

### PR DESCRIPTION
**Pull request by olalonde**

### CODE REVIEW

File: serfclient/connection.py [hunk 1]
```diff
@@ -2,6 +2,7 @@
 import sys
 import msgpack
 import resource
+import queue
```
Comment (KushalP) [inline serfclient/connection.py]: Is this being used here? It looks like it's breaking the build in a few places.

Comment (olalonde) [inline serfclient/connection.py]: Oops, it's not, removing it.

**Comment by olalonde** [comment]
@KushalP do you know how I would fix https://travis-ci.org/KushalP/serfclient-py/jobs/258695208 ? I just been doing Python for a few weeks and this looks like a backwards compatibility issue?

**Comment by KushalP** [comment]
@olalonde you're doing great so far! Python 2 and 3 have a few odd differences and it's worth learning them and being able to spot them over time. In the case of these tests, it's worth picking one of the failing tests in question (2.6 or 3.3) and working from there.

# pdb

A useful tool to learn to use is `pdb`: https://docs.python.org/2/library/pdb.html

With it, you can set a trace point and when you run some Python code a REPL will appear allowing you to get at any of the values that are in scope. Here's a small example:

```python
import pdb

a = 1
pdb.set_trace()
# I can run this code, and a REPL will appear here, allowing me to read the value of `a`.
```

You can use this method, at the line *before* your test fails, to examine what the state of each of your values are, so that you can better debug this issue.

# -v (verbose mode)

As [test test line](https://travis-ci.org/KushalP/serfclient-py/jobs/258695208#L5660) in question says, you can use `-v` when running the tests to see the verbose failure reasons.

# virtualenv

To test multiple versions of Python locally, a common practice is to use [virtualenv](https://virtualenv.pypa.io/en/stable/). It's a package that allows you to (locally) set up multiple Python versions and change between them.

That should help you better understand what's going on here. Let me know if you're still having trouble getting to the bottom of this.

**Comment by olalonde** [comment]
Thanks for the tips :) I fixed the tests that were failing.

**Comment by KushalP** [comment]
Excellent work! Can you squash the fix into the previous commit please? Then I'll be happy to merge this and get a release out.

**Comment by olalonde** [comment]
Just did, thanks 👍

**Comment by KushalP** [comment]
Thank you! I'll try to get a release made later this week. I'm not near my machine which has the relevant GPG keys.

**Pull request merged**